src/main/instantReloadProcess.ts

```diff
@@ -60,6 +59,28 @@ const handleCreate = async (payload) => {
 
 	const instantReloadUrl: string = await new Promise((resolve) => {
 		instantReloadInstance.init({
+			logPrefix: 'Instant Reload',
+			// @ts-expect-error: Temporarily ignoring this error until
+			// upstream fix is available:
+			//     https://github.com/BrowserSync/browser-sync/pull/2093
+			notify: {
+				styles: {
+					display: 'none',
+					padding: '15px',
+					fontFamily: 'sans-serif',
+					position: 'fixed',
+					fontSize: '1em',
+					zIndex: 9999,
+					right: '1em',
+					top: '1em',
```

Comment:
```suggestion
					zIndex: 1000000,
					right: 0,
					top: 0,
```

Worth tweaking the zIndex and making right and top 0 to better mimic the original positioning from the forked repo?

Otherwise we see a weird gap to the right and top when there's no WP admin bar:

| Original positioning | Current PR | This PR with right/top zeroed |
| - | - | - |
| ![original-notification-styling](https://github.com/user-attachments/assets/d3d7be07-1e04-4de9-ac7b-b588ffa68fc7) | ![notification-styling-this-pr](https://github.com/user-attachments/assets/c0d89c33-247c-4bd5-b552-13227cead734) | ![notification-styling-zeroed](https://github.com/user-attachments/assets/da35da8f-dd62-41c5-a63b-33975a007617) |

The zIndex tweak is to make sure the notification appears above the WP admin bar:

| With admin bar | Without admin bar |
| - | - |
| ![with-admin-bar](https://github.com/user-attachments/assets/d3dd7afe-30de-4936-9cdf-f93bce6f62a4) | ![without-admin-bar](https://github.com/user-attachments/assets/cefe36d6-cbf7-4889-b8bb-73a0306f3658) |

Which also helps to match the original:

![original-z-index](https://github.com/user-attachments/assets/b736a7eb-609e-48e1-9042-7eb0373df8a8)